app/scripts/modules/kubernetes/src/v2/manifest/json/JsonEditor.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import AceEditor from 'react-ace';
+import { $log } from 'ngimport';
+
+import 'brace/theme/textmate';
+import 'brace/mode/json';
+import './jsonEditor.less';
+
+export interface IJsonEditorProps {
+  value: string;
+  onChange(raw: string, manifest: any): void;
```

Comment:
Using the word `manifest` here for the yaml editor was a mistake on my part - maybe `obj` or something? (Doesn't seem like this component necessarily has to be used for editing manifests)

Reply:
Good call, replaced with `obj` here and in the `YamlEditor`.